charts/nginx-gateway-fabric/README.md

```diff
@@ -1,7 +1,7 @@
 
 # NGINX Gateway Fabric Helm Chart
 
-![Version: 1.2.0](https://img.shields.io/badge/Version-1.2.0-informational?style=flat-square) ![AppVersion: edge](https://img.shields.io/badge/AppVersion-edge-informational?style=flat-square)
+![Version: 1.3.0](https://img.shields.io/badge/Version-1.3.0-informational?style=flat-square) ![AppVersion: 1.3.0](https://img.shields.io/badge/AppVersion-1.3.0-informational?style=flat-square)
```

Comment:
Since we're updating version strings in the main branch, we really shouldn't merge this until we are ready to cut the release tag.

Reply:
This is PR is updating the release branch only

Reply:
Oh I see.